pkg/reconcile/pipeline/infinispan/handler/provision/statefulsets.go

```diff
@@ -127,6 +127,7 @@ func ClusterStatefulSetSpec(statefulSetName string, i *ispnv1.Infinispan, ctx pi
 							MountPath: DataMountPath,
 						}},
 					}},
+					PriorityClassName: i.PriorityClassName(),
```

Comment:
Would it be possible to write some kind of unit test for the value propagation to StatefulSet? Unfortunately functional test is not reasonably applicable here.

Reply:
Good point, I have added a unit test.